invoices/invoice_expiry_watcher_test.go

```diff
@@ -0,0 +1,46 @@
+package invoices
+
+import (
+	"testing"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lntypes"
+)
+
+// Tests that invoices are canceled after expiration.
+func TestInvoiceExpiry(t *testing.T) {
```

Comment:
Other cases that could be exercised (non-exhaustive list):
  * empty queue, no invoices cancelled
  * full queue, no invoices cancelled as not yet past expiry 
  * populated queue, add new invoice has already expired in the past 
  * populated queue, add new invoice, should be woken up to cancel

Reply:
Will add a few more tests in an upcoming commit.